HCK24_2025_Janelia_NDRH/README.md

```diff
@@ -0,0 +1,146 @@
+[//]: # (<img alt="NeuroDataReHack banner" src="images/NDRH2025_banner.png">)
+
+# NeuroDataReHack 2025
+## *Generate new insights from existing neurophysiology data through secondary analysis*
+
+[//]: # (* [Report]&#40;#report&#41;)
+* [Dates and Location](#dates-and-location)
+* [Objective](#objective)
+* [Logistics](#logistics)
+* [Organizing Committee](#organizing-committee)
+* [Resources](#resources)
+* [What to bring?](#what-to-bring)
+* [Schedule](#schedule)
+* [Disclaimer](#disclaimer)
+* [Eligibility](#eligibility)
+* [Application](#application)
+
+[//]: # (## Report)
+
+[//]: # ()
+[//]: # (The final report for NeuroDataReHack 2024 is now available online at [[PDF]&#40;https://drive.usercontent.google.com/u/0/uc?id=1QYw9jmIpKPXPMLmhTnVIIWgbWHL5tPS3&export=download&#41;].)
+
+
+
+## Dates and Location
+
+- **Dates:** July 14-18, 2025
+- **Location:** [HHMI Janelia Research Campus](https://www.janelia.org/), 19700 Helix Dr, Ashburn, VA 20147,
+  [Openstreetmap](https://www.openstreetmap.org/?mlat=39.0708&mlon=-77.4655#map=14/39.0708/-77.4655)
+  [Directions](https://www.janelia.org/directions)
+
+## Objective
+[The DANDI Archive](http://dandiarchive.org) now has 300+ neurophysiology datasets in the 
+[Neurodata Without Borders](http://nwb.org) format  spanning many species, brain  areas, task types, and imaging 
+modalities. These include high-value datasets, e.g. from [Allen Institute OpenScope](https://dandiarchive.org/dandiset/search?search=openscope), the [MICrONS project](https://dandiarchive.org/dandiset/000402), and the 
+[International Brain Laboratory Brain Wide Map](https://dandiarchive.org/dandiset/000409), as well as diverse contributions from neuroscience labs around the world. In this 
+workshop, we will teach attendees about the open neurophysiology datasets available on the DANDI Archive and train 
+them on how to maximally utilize the archive and the NWB standard to incorporate existing data into their scientific 
+workflows. Feedback from attendees will be used to improve the software and data standard to better enable 
+reanalysis workflows.
+
+
+Prior to the workshop, we are organizing [Open Neurodata Showcase](../HCK23_2025_OpenNeurodataShowcase/README.md) where
+attendees can meet the contributors behind the neurophysiology datasets and explore virtual posters. Visit the
+[event page](../HCK23_2025_OpenNeurodataShowcase/README.md#application) to sign up and read more about this feature
+event.
```

Comment:
Let's add it in a different PR. I want to make sure this gets up so we can begin promoting it. This needs a lot more lead time since it is requires application and review.

Reply:
sounds good!